features/htcondor/client/ce.pan

```diff
@@ -0,0 +1,50 @@
+unique template features/htcondor/client/ce;
+
+variable CONDOR_CONFIG={
```

Comment:
I may be wrong but I would say that '?=' is not an option here. The variable CONDOR_CONFIG is not necessarily defined here (actually it is very likely to be already defined at this point). It is the general configuration variable for all htcondor related stuff. This template just adds some parameters to the variable (i.e. adds the 'submit' standard file to the list of config files and initialize the relative options). Putting ?= will make the template ineffective, will it not?